pkg/sfu/downtrack.go

```diff
@@ -459,6 +463,17 @@ func (d *DownTrack) Bind(t webrtc.TrackLocalContext) (webrtc.RTPCodecParameters,
 		return codec, nil
 	}
 
+	// Bind is called under RTPSender.mu lock, call the RTPSender.GetParameters in goroutine to avoid deadlock
+	go func() {
```

Comment:
revert the change since it is not the issue and we can get rtp header extension earlier